packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,32 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaDivider, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaDivider, VaButton },
+  methods: {
+    change (color) {
+      useColors().setColors({ danger: color })
+    },
+  },
+  template: `
+    <div class="mb-6">
+      <a class="title title--va-danger">
```

Comment:
We test here css variables, I don't remember what is title class and where can I find it... How about we actually use `var(--va-danger)` here and simplify this story?